tests/NLog.UnitTests/Targets/LogReceiverWebServiceTargetTests.cs

```diff
@@ -190,10 +190,18 @@ public void NoEmptyEventLists()
             {
                 Name = "NoEmptyEventLists_wrapper"
             };
-            asyncTarget.Initialize(configuration);
-            asyncTarget.WriteAsyncLogEvents(new[] { LogEventInfo.Create(LogLevel.Info, "logger1", "message1").WithContinuation(ex => { }) });
-            Thread.Sleep(1000);
-            Assert.Equal(1, target.SendCount);
+            try
+            {
+                asyncTarget.Initialize(configuration);
+                asyncTarget.WriteAsyncLogEvents(new[] { LogEventInfo.Create(LogLevel.Info, "logger1", "message1").WithContinuation(ex => { }) });
+                Thread.Sleep(1000);
+                Assert.Equal(1, target.SendCount);
+            }
+            finally
+            {
+                asyncTarget.Close();
```

Comment:
This is similar to the code in `AsyncTargetWrapperTests`. Running the tests (with the old code) through the debugger shows that the `AsyncTargetWrapper` is not closed. 

[https://github.com/NLog/NLog/blob/master/tests/NLog.UnitTests/NLogTestBase.cs#L62](https://github.com/NLog/NLog/blob/master/tests/NLog.UnitTests/NLogTestBase.cs#L62) does not take care of this, because in these tests the `AsyncTargetWrapper` is not in the configuration.

After the change debugging the tests confirms that the `AsyncTargetWrapper` is closed.